pkg/services/sqlstore/migrations/migrations.go

```diff
@@ -82,7 +82,6 @@ func (*OSSMigrations) AddMigration(mg *Migrator) {
 	accesscontrol.AddManagedPermissionsMigration(mg, accesscontrol.ManagedPermissionsMigrationID)
 	accesscontrol.AddManagedFolderAlertActionsMigration(mg)
 	accesscontrol.AddActionNameMigrator(mg)
-	addPlaylistUIDMigration(mg)
```

Comment:
note: this does not remove the migration -- the same migrations are now added in `addPlaylistMigrations`

Reply:
I know this doesn't remove the migration, but we must not change the migrations list! It is additive only - that's set out in our contribution guidelines and changing past migrations can cause unexpected behavior in existing instances: https://github.com/grafana/grafana/blob/main/contribute/backend/database.md#migrations